managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -594,4 +593,6 @@ void asyncSetProperties(Map<String, String> properties, final AsyncCallbacks.Upd
      * Get the ManagedLedgerInterceptor for ManagedLedger.
      * */
     ManagedLedgerInterceptor getManagedLedgerInterceptor();
+
+    CompletableFuture<LedgerMetadata> getRawLedgerMetadata(long ledgerId);
```

Comment:
Can you explain why do we need this method? Also, add a comment to this method?

Reply:
I would like to see if we can avoid adding `LedgerMetadata` here. Because `LedgerMetadata` is a specific interface for bookkeeper. We should avoid binding the metadata format to bookkeeper's ledger metadata. This would simplify tiered storage integration.

Reply:
We use `LedgerMetadata` because we need to use tiered storage to implement a `org.apache.bookkeeper.client.api.ReadHandle`, which is also a specific interface for bookkeeper, I think it's not avoidable.

Reply:
> Can you explain why do we need this method? Also, add a comment to this method?

Former offloader get a `ReadHandle` from `ManagedLedger`, and get `LedgerMetadata` from `ReadHandle`, our new offloader does not hold any `ReadHandle` but it still needs `LedgerMetadata` to make the block index

Reply:
Actually, I want to name this method to `getLedgerMetadata`, but a method with this name already exists https://github.com/apache/pulsar/blob/138390ca27a49682cb9d0a0ff256c01e1bc347e3/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java#L1685 
Maybe it's better to name the existing method `getLedgerMetadataStr`